bin/_test-helpers.sh

```diff
@@ -468,6 +468,10 @@ run_cni-calico-deep_test() {
   run_test "$test_directory/deep/..." --cni
 }
 
+run_rsa-ca_test() {
+  run_test "$test_directory/rsa-ca/..."
+}
+
```

Comment:
For this test to be triggered, can you add `rsa-ca` to `default_test_names`?
Also it'd be good to add it to the `integration_test` array in both the `integration_tests.yml` and `release.yml` workflows.

Reply:
Thanks for the patience :)

The `too-weak-rsa-anchor*` certs were intended to test that the RSA signature of the trust anchor was strong enough. Nevertheless, I see that there are no tests regarding the strength of the chain. Does it make sense to add these? Or better just remove them?

As we just provide the public key of the trust anchor, it would be also dirty creating a fake Cred without a Key just for the sake of testing it. WDYT?

Regarding the DCO, I added the last line on the commit message. Is that enough?
Otherwise, if I need to also add something here, I  agree to the DCO for all the commits in this PR. :)

Reply:
I think your test you already added is enough, so you can remove those unused files :+1: 
I don't know why GH didn't accept the DCO, so I've just had it manually pass.